src/BasketBundle/Controller/BasketController.php

```diff
@@ -47,7 +49,7 @@ public function indexAction($form = null)
         ]);
 
         // always validate the basket
-        if (!$form->isBound()) {
+        if (!$form->isSubmitted()) {
```

Comment:
This change is not related to this PR, is this because of Sf3 compat?

Reply:
I have already fixed these deprecations in previous PR, but missed this one. And yes - this is for SF 3.